web/client-api/src/main/java/io/deephaven/web/client/api/CoreClient.java

```diff
@@ -93,6 +134,14 @@ public Promise<UserInfo> getUserInfo() {
     }
 
     public JsStorageService getStorageService() {
-        return new JsStorageService(connection.get());
+        return new JsStorageService(ideConnection.connection.get());
+    }
+
+    public Promise<IdeConnection> getAsIdeConnection() {
+        return Promise.resolve(ideConnection);
```

Comment:
Because I'm trying to allow for that not always being the way that this will work - easier to just always be async and adding a slight delay for a one-time call, than try to add it later?

My guess is that we will eventually default to assuming that the server isn't a worker (but only config/storage/both), and we will need to confirm that it can be used as a worker.